install_config/configuring_openstack.adoc

```diff
@@ -13,8 +13,7 @@ toc::[]
 == Overview
```

Comment:
In talking this over with @TheDiemer he pointed out that the titles / section headers should read: 

Configuring OpenShift Enterprise [COMPONENT] for PROVIDER/PLATFORM

As its more accurate to what we are "configuration"